### ISSUE TITLE
Pick List, Pick List Group and Pick List Item - Ensure all interactive functionality is operable with the keyboard - (2036570117)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#5594]
### Module: 
 24 Pick List, Pick List Group and Pick List Item 

### Violation: 
 Ensure all interactive functionality is operable with the keyboard 

![image](https://user-images.githubusercontent.com/5023024/202743637-d7e303ad-02d7-4ef0-98c8-8e868ccbb4b8.png)

### WCAG Reference: 
- [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html) 

### Instance ID: 
 2036570117 

### Severity: 
 8 

### Description: 
 [Issue]
There are interactive controls that cannot be navigated to and operated with the keyboard alone. The focus moves to the first interactive control only using TAB.
Examples include:
- pick-list-item below "Test 1. Pick List", "Test 2. Removable Pick List Items"," Test 4. Heading Level", etc.

[User Impact]
Keyboard-only users will be prevented from accessing the functionality provided by these controls.

[Code Reference] 
 ```
<label aria-label="River" class="label" tabindex="0"><div aria-checked="false" class="text-container" role="menuitemcheckbox"><span class="title">River</span><span class="description">A large natural stream of water flowing in a channel to the sea, a lake, or another such stream.</span></div></label>
 ``` 

### Note: 
 [Suggestion]
Ensure all interactive functionality is operable with the keyboard. The best way to accomplish this is by using appropriate native controls, which come with keyboard functionality built in.

If using a native control is not possible, the control must have tabindex="0" and have JavaScript event listeners that allow the control to be activated when Enter is pressed.

[Compliant Code Example] 
 ```
N/A
 ```
 

### Media Type: 
 Keyboard Accessibility 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
This should be addressed in the updated `list` component.

**Comment by driskull** [comment]
Yes we can ignore this one for now.

**Comment by geospatialem** [comment]
Going to add to the post 1.0 project so we can revisit post-1.0 to confirm if the updated component fulfills this issue.

**Comment by geospatialem** [comment]
Closing as the `pick-list` components are deprecated and functionality will not be supported at this time.
